customizations.md

```diff
@@ -115,33 +115,43 @@ In the second option, a fleet manager would configure the map file on ECUs such
+Another way to provide dynamic directions is to use the custom field of the Targets metadata file. This field provides the option to include custom inputs to individual ECUs. Using the custom field is an especially good option for managing small variations in the existing image. For example, a compilation flag to enable a navigation feature might be set on some ECUs, but not on others. The custom field could contain dynamic directions, and additional subfields would help determine for which ECUs the direction is intended. In the flag example above, the Director can put the ECU ID and the flag into the custom field so the flag will be used during the installation process only on that particular ECU. This custom field can then be included in the Targets metadata received by all ECUs. The intended ECU would be able to check for this flag and use it during an installation or update to enable the navigation system.
 
-Using this method of providing dynamic directions offers several ways to secure the system. The Targets metadata is created by the Director, validated using the Timestamp, and stored in the Image repository. As an added protection against a compromise of the Director role, the custom field could also be included in the Targets metadata file on the Image repository. This option works best if there is only a small set of known customizations. However, if there are multiple customization possibilities, the better option would be to store the Targets metadata without the custom field in the Image repository, and keep the custom field configured separately and signed by the Director.
+However, note that using this method of providing dynamic directions means that a compromise of the Director repository might be able to cause ECUs to misconfigure their images. One way to mitigate this risk would be to require the Image repository to sign off on exactly the same directions using its own custom Targets metadata. but this is difficult to achieve considering that such directions are supposed to be dynamic in the first place. Therefore, proceed, if necessary, with caution.
```

Comment:
```suggestion
However, note that using this method of providing dynamic directions means that a compromise of the Director repository might be able to cause ECUs to misconfigure their images. One way to mitigate this risk would be to require the Image repository to sign off on exactly the same directions using its own custom Targets metadata. However, this is difficult to achieve considering that such directions are supposed to be dynamic in the first place. Therefore, proceed, if necessary, with caution.
```

Reply:
Looks good to me. I'm going to commit and merge.